### ISSUE TITLE
How I can install ?

### ISSUE THREAD

**Issue by mauricioA0** [zhuyifan/Carpooling#2]
Hi 

Is the project free ?

How I can install the application ?

Thanks

**Comment by zhuyifan** [comment]
Yeah it’s free. You need to download WeChat in you phone first and also need WeChat IDE to install it.

https://mp.weixin.qq.com/debug/wxadoc/dev/devtools/download.html?t=201818

You can find more information of WeChat mini program in https://mp.weixin.qq.com/debug/wxadoc/dev/

You could use chrome to translate the website.

Good luck!
